database/index.js

```diff
@@ -38,7 +38,7 @@ const getBonusItem = (title, callback) => {
 
 // Get song to be played upon selection
 const getSong = (trackName, callback) => {
-  Soundtrack.findOne({ 'bonus_info.tracklist.name': trackName}, {'bonus_info.tracklist.$': 1, _id: 0}, (err, data) => {
+  Soundtrack.findOne({ 'bonus_info.tracklist.name': trackName }, { 'bonus_info.tracklist.$': 1, _id: 0 }, (err, data) => {
```

Comment:
I just added spaces here for consistency.

Reply:
ok yeah that's a bit more readable ✓